backend/server/src/main/java/ai/verta/modeldb/experimentRun/subtypes/VersionInputHandler.java

```diff
@@ -283,7 +283,7 @@ private void insertVersioningInput(
                 + " VALUES (:repository_id, :commit, :versioning_key, :versioning_location, :entity_type, :versioning_blob_type, :blob_hash, :entityId)",
             App.getInstance().mdbConfig.getDatabase().getRdbConfiguration().isMssql()
                 ? "\"commit\""
-                : "commit",
+                : COMMIT_QUERY_PARAM,
```

Comment:
would revert this. Just makes things more confusing

Reply:
Reverted